### ISSUE TITLE
File-like Interface

### ISSUE THREAD

**Issue by CtrlC-Root** [branan/puppet-module-s3file#3]
It would be kind of nice and make the type more useful if it had a similar set of parameters to the built in file type (http://docs.puppetlabs.com/references/latest/type.html#file). This shouldn't even be that hard to do, since you could declare an instance of the file{} type and just pass most of the parameter values into it directly. This isn't critical and I'm not waiting on it or anything, but it would be nice to have.

**Comment by mlehner616** [comment]
+1 (esp. mode/owner/group)
